projectroles/plugins.py

```diff
@@ -238,6 +238,16 @@ def revert_project_archive(
         # TODO: Implement this in your app plugin
         pass
 
+    def callable_function(self, project, user):
```

Comment:
We can remove this from here, because this assumes a single function in every plugin. It'll be enough if we describe in the docs how these methods can be implemented.